sources/packages/backend/libs/integrations/src/institution-integration/ier12-integration/ier12-file-detail.ts

```diff
@@ -37,51 +37,80 @@ export class IER12FileDetail implements IER12FileLine {
   disbursementSchedules: DisbursementSchedule[];
 
   getFixedFormat(): string {
-    const disbursementValuesTotal: DisbursementValue[] =
-      this.disbursementSchedules.flatMap(
-        (disbursement) => disbursement.disbursementValues,
+    const records = this.disbursementSchedules.map((disbursement) => {
+      const record = new StringBuilder();
+      record.appendWithStartFiller(this.assessmentId, 10, NUMBER_FILLER);
+      record.appendWithStartFiller(disbursement.id, 10, NUMBER_FILLER);
+      record.appendWithStartFiller(this.applicationNumber, 10, NUMBER_FILLER);
```

Comment:
why using number filler here? can the application number be less than 10 characters?